packages/components/src/Loading/LoadingSpinner/_docs/LoadingSpinner.stories.tsx

```diff
@@ -25,12 +25,27 @@ export const Playground: Story = {
 }
 
 export const ClassNameOverride: Story = {
-  args: { classNameOverride: "text-green-400" },
+  render: args => (
+    <>
+      <LoadingSpinner {...args} classNameOverride="text-green-400" />
```

Comment:
Curious about the need to increase these examples?

Reply:
I was aligning this to the examples Jess has in Figma, figured while we probably don't want to introduce a color prop we can at least show the one that match

Reply:
Hmm...not really necessary imo to completely match the design examples unless our API is strictly limited to their selection (eg. variants). This story is showcasing the `classNameOverride` prop, and actually in this case, you're preventing the controls from working 😅

Reply:
Isn't this story more to showcase in our documentation, ie:

![Screenshot 2024-10-10 at 11 35 15 AM](https://github.com/user-attachments/assets/6afd2c35-92b9-4db7-afcd-3c0645f1f184)

To our consumer the controls never did anything (at least on the docs page).

I suppose the question I'm wondering is, are we meant to have standard spinner colors, since they have been specifically designed?

Reply:
Maybe a separate story to delineate the difference between just classNameOverride and colors if they're needed? My main concern stems from have mild color variances in spinners

Reply:
I'd say that standard colours aren't needed. Using `classNameOverride` is the correct pattern.

The docs page doesn't show the controls for that story, but the individual story does :)

But yeah, I guess looking at that screenshot, I suppose showcasing a few more is nice 🤔 Perhaps change the order so that the one that can be changed by controls is the first one? Or move the spread args to the end so they can all be overridden by the controls?

That being said, color isn't the only use case for this (eg. margin, display), so I wonder if it would seem too specific and confusing only showing multiple colours 🤔 I'll leave the decision to you :)

Reply:
mmm I might check in with Jess to see how important those colors are to document, it could be a null issue in which case 🤷 but yeah to your point maybe leaving the classNameOverride story clean is good since it doesn't technically just to color

Reply:
Cool, so just chatted with Jess. They do have a set of colors that they use, but I said I wouldn't add scope to the ticket. I've create an additional story to be a little more explicit on the colors but kept it separate from the classNameOverride story